app/controllers/api/v1/classrooms_controller.rb

```diff
@@ -72,7 +72,7 @@ def mail_worksheets(work_group)
   def mail_worksheets_to_teacher(work_groups)
     DemoMailer.with(
       teacher: work_groups.first.classroom.teacher,
-      work_groups: work_groups.deep_pluck(users: :name, worksheets: :image_url)
+      work_groups: work_groups.deep_pluck(users: %i[name pen_colour], worksheets: :image_url)
```

Comment:
actually in another PR I was trying to get rid of deep pluck as it's going to be deprecated and apparently forgot about this one
can you figure out a workaround? :)